zeppelin-web/src/app/home/home.html

```diff
@@ -27,7 +27,7 @@ <h1 class="box-heading" id="welcome">
         <h4>Notebook</h4>
 
         <div>
-          <h5><a href="javascript:void(0);" ng-click="home.websocketMsgSrv.createNotebook()" style="text-decoration: none;">
+           <h5><a href="javascript:void(0);" data-toggle="modal" data-target="#noteNameModal" style="text-decoration: none;">
```

Comment:
I think this line is why i see XMLHttpRequest error. `javascript:void(0);` was removed from menu, but not removed from home screen.